code/src/reducers/store.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable linebreak-style */
+import { createSlice } from '@reduxjs/toolkit';
+import { ui } from './ui';
+
+
+const initialState = {
+    username: "",
+    currentDirection: [],
+     
+}
+
+export const game = createSlice({
+    name:'game', 
+    initialState,
+    reducers: {
+        setUserName: (store, actions) => {
+            store.username = actions.payload
+            // Next step is to connect username to unique ID
+        },
+        setCurrentDirection: (store, actions) => {
```

Comment:
you could export the action const {setCurrentDirection} = game.actions (this game is your gameSlice, so it's maybe better to name it gameSlice to indicate the purpose

once you export your actions, you don't have to type action.seCurrentDirection when you dispatch it